content/en/quickstart/quickstart-ci.md

```diff
@@ -0,0 +1,195 @@
+
+```console
+name: signing_files
+# This will trigger the workflow to run when commits are pushed to the main branch. This is easy for testing purposes, but for your final workflow use whatever event or schedule makes sense for your project.
+on:
+    push:
+        branches: [ main ]
+jobs:
+  signing_files:
+    runs-on: ubuntu-latest
+    # 'id-token' needs write permission to retrieve the OIDC token, which is required for authentication.
+    permissions:
+      id-token: write
+    steps:
+        # This step ensures that your project is available in the workflow environment.
+        - uses: actions/checkout@v3
+        # This step uses 'gh-action-sigstore-python' to sign the file designated in the inputs field.
+        - uses: sigstore/gh-action-sigstore-python@v3.0.0
+          with:
+            inputs: to_be_signed.txt
+```
+
+When run, this workflow returns the ephemeral certificate used to sign the file, as well as the index for the transparency log entry.
+
+### Verifying your signed files
+
+The `gh-action-sigstore-python` GitHub Action includes an option to verify your generated signature. This is optional but a great way to understand the GitHub Action as you are integrating it into your CI for the first time. To verify the signature you just created, set the `verify` setting to true and include your expected `verify-cert-identity` and `verify-oidc-issuer` settings.
+
+```console
+        - uses: sigstore/gh-action-sigstore-python@v3.0.0
+          with:
+            inputs: to_be_signed.txt
+            verify: true
+            verify-cert-identity: https://github.com/USERNAME/REPOSITORY_NAME/.github/workflows/WORKFLOW_NAME@refs/heads/BRANCH_NAME
+            verify-oidc-issuer: https://token.actions.githubusercontent.com
+```
+
+## Using Cosign within your CI system
+
+If you need functionality beyond simple signing of files and blobs, you can use the [`consign-installer` GitHub Action](https://github.com/marketplace/actions/cosign-installer) to [integrate Sigstore into your CI system](#installing-cosign-on-your-ci). This quickstart covers:
+
+- How to [sign and verify a container image](#signing-and-verifying-a-container-image) using your CI system
+- How to [sign](#signing-a-blob) and [verify](#verifying-a-blob) a blob using `consign-installer`
+
+### Installing Cosign on your CI
+
+The following workflow will install Cosign into your workflow environment.
+
+```console
+name: install-cosign-and-use
+on:
+  # This will trigger the workflow to run when commits are pushed to the main branch. This is easy for testing purposes, but for your final workflow use whatever event or schedule makes sense for your project.
+   push:
+    branches: [ main ]
+# No special permissions are required to install cosign, but `id-token: write` is needed to sign with your workflow identity.
+permissions:
+  id-token: write
```

Comment:
Nitpick: This could be moved into the `install-cosign-and-use` job to minimize permission sharing (per-job instead of per-workflow). Mostly a moot point in this context since it's only a single-job workflow, though!